dashboard/export_scores.R

```diff
@@ -0,0 +1,42 @@
+source('./common.R')
+
+create_export_df = function(scoreDf, targetVariable, forecasters, horizon, loc) {
+    signalFilter = CASE_FILTER
+    if (targetVariable == "Deaths") {
+      signalFilter = DEATH_FILTER
+    }
+    scoreDf = scoreDf %>%
+      filter(signal == signalFilter) %>%
+      filter(ahead %in% horizon) %>%
+      filter(forecaster %in% forecasters)
+    if (loc != TOTAL_LOCATIONS) {
+      scoreDf = scoreDf %>% filter(geo_value == tolower(loc))
```

Comment:
I'm wondering if we should include somewhere what locations are included when the user has selected total locations (not all of the locations are included - it is based on what forecasters are selected and which locations they have in common). But maybe that is not necessary...

Reply:
how is this information currently shown in the user interface?

Reply:
Currently it is shown at the bottom like this: 
<img width="883" alt="Screen Shot 2021-06-30 at 12 42 30 PM" src="https://user-images.githubusercontent.com/14190352/123999611-b6373c00-d9a0-11eb-8b30-9503629a6189.png">